fe/fe-core/src/main/java/com/starrocks/sql/optimizer/CTEUtils.java

```diff
@@ -144,15 +142,6 @@ private static void collectCteConsume(Group root, OptimizerContext context, bool
             if (collectCosts && !expression.getInputs().isEmpty()) {
                 context.getCteContext().addCTEConsumeInlineCost(consume.getCteId(),
                         calculateStatistics(expression.getInputs().get(0), context).getComputeSize());
-            } else if (collectCosts && expression.getInputs().isEmpty()) {
```

Comment:
Why remove this

Reply:
Unuse, will enforce use CTE, don't need mock  consume costs